src/database/index.tsx

```diff
@@ -0,0 +1,115 @@
+import * as firebase from "firebase";
+import "firebase/auth";
+import "firebase/database";
+import { pickBy } from "lodash";
+
+import createAuth0Client from "@auth0/auth0-spa-js";
+
+import config from "../auth/auth_config.json";
+import { EpidemicModelUpdate } from '../impact-dashboard/EpidemicModelContext';
+
+// As long as there is just one Auth0 config, this endpoint will work with any environment (local, prod, etc.).
+const tokenExchangeEndpoint =
```

Comment:
For the time being, I'm assuming we're just going to have one DB to serve all tiers?  Meaning if I'm logged in and doing development locally I'll be reading/writing to the same DB instance as when I'm logged into production?

Reply:
Right, that's the case for the time being.